packages/playwright-test/src/util.ts

```diff
@@ -294,3 +294,5 @@ export async function normalizeAndSaveAttachment(outputPath: string, name: strin
     return { name, contentType, body: typeof options.body === 'string' ? Buffer.from(options.body) : options.body };
   }
 }
+
+export const pluginLogger = (plugin: TestPlugin) => debug(`pw:plugin:${plugin.name}`);
```

Comment:
This is keeping it simple. I almost wrapped our plugins with a Proxy so we would log accesses to the interesting methods, but that became silly.